test/runners/julia_spec.js

```diff
@@ -23,7 +23,7 @@ describe('julia runner', function() {
         `,
         testFramework: 'Test'
       }, function(buffer) {
-        expect(buffer.stdout).to.equal('("<PASSED::>Test Passed","<:LF:>\\n")');
+        expect(buffer.stdout).to.equal('("<PASSED::>Test Passed", "<:LF:>\\n")');
```

Comment:
Thanks. I asked because you authored that a year ago? (https://github.com/Codewars/codewars-runner-cli/commit/c9fc0f22c615515c62ac4f0d9f7074549c63f845)
Apparently, you also wrote the test framework.
I'm pointing this out because codewars.com cannot use it unless the output format is correct.